orochi/client.py

```diff
@@ -685,6 +705,12 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            f = open(expanduser("~") + '/.orochi.current_song.log', 'w')
+            f.write(u' '.join((track_name, '|', track_performer)).encode('utf-8').strip())
```

Comment:
you can always do $(cat current_song.txt | cut -d'|' -f1) to get a specific attribute as well right ? 

a single line helps me do a simple cat if i need all of it to be shown in my status bar..